### ISSUE TITLE
Bug: Class constant as default function value in Helper translated to wrong namespace in `...TesterActions.php`

### ISSUE THREAD

**Issue by ThomasLandauer** [Codeception/Codeception#6024]
I'm not sure if this will automatically be fixed through https://github.com/Codeception/Codeception/issues/5727

When I have this in my `Helper/Acceptance.php`:
```php
use App\Entity\Foo;
public function whatever(string $input = Foo::BAR) { }
```
... the generated method in `AcceptanceTesterAction.php` looks like this:
```php
public function whatever($whatever = App\Entity\Foo::BAR) { }
```
...which leads to this error message:
> [Error] Class 'App\Tests\_generated\App\Entity\Foo' not found

... cause `App\Entity\Foo` is resolved *relatively* to the `AcceptanceTesterAction`s namespace `App\Tests\_generated`.

### Details

* Codeception version: 4.1.9

**Comment by javabudd** [comment]
confirmed, after updating to 4.1.9 this is happening to my code as well

**Comment by Naktibalda** [comment]
Fixed by #6016 , released as 4.1.10

**Comment by mwi-gofore** [comment]
Actually, the fix only works for cases where the constant is defined in the same class. If the constant comes from another class, it still fails :(
